### ISSUE TITLE
assetseed: Update to work with the current seed format.

### ISSUE THREAD

**Issue by JoeGruffins** [decred/dcrdex#3108]
We have an app at /client/cmd/assetseed.  It only works with the old hex format but should work for both that one and the newer 15 word seed.

I wonder if we could also spit out something for bitcoin clones that a user can use to import into whatever core wallet. eth can already do this from the UI I believe. mentioned here https://github.com/decred/dcrdex/issues/3105#issuecomment-2516297259

Does the seed work with Bitcoin core?

**Comment by dev-warrior777** [comment]
Anything to do with [this #3055](https://github.com/decred/dcrdex/issues/3055) Windows specific issue? Fix [#3055](https://github.com/decred/dcrdex/pull/3056) has been merged but not sure in included release

**Comment by freQniK** [comment]
Nice. So I could do asset=2 with the Bison seed and it should give me the raw hex private key of the litecoin wallet?

I don't need this now as I switched my Litecoin wallet to my Litecoin core node, but would be good to know for other users.

**Comment by JoeGruffins** [comment]
No, I looked into it and the seed is not compatible with bitcoin/litecoin core, sadly. I don't think there's anything we can do to restore internal wallets with other software.

**Comment by freQniK** [comment]
> No, I looked into it and the seed is not compatible with bitcoin/litecoin core, sadly. I don't think there's anything we can do to restore internal wallets with other software.

Ahh. This is probably why when I wrote a python script to produce the private key from the Bison seed, I couldn't get it to produce the same wallet in other Litecoin wallets like electrum. I kept thinking I was doing something wrong.
